Tabloid/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
```

Comment:
Should this controller be auth protected? do we need any other auth checks on the route?

Reply:
Taken care of @heymonicakay